dirty.rb

```diff
@@ -0,0 +1,38 @@
+a =     'b'
+eval '1+1'
+"#{1}"
+begin
+  foo
+rescue
+end
+begin
+begin
+begin
+begin
+  fooaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
__[development][rubocop]__
* Line is too long. [245/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)